source/amazon/services/prerequisites/considerations.rst

```diff
@@ -22,8 +27,30 @@ If the S3 bucket contains a long history of logs and its directory structure is
 Older logs
 ----------
 
-The ``aws-s3`` Wazuh module only looks for new logs based upon the key for last processed log object, which includes the datetime stamp. If older logs are loaded into the S3 bucket or the ``only_logs_after`` option date is set to a datetime earlier than previous executions of the module, the older log files will be ignored and not ingested into Wazuh.
+The ``aws-s3`` Wazuh module only looks for new logs in buckets based upon the key of the last processed log object, which includes the datetime stamp. If older logs are loaded into the S3 bucket or the ``only_logs_after`` option date is set to a datetime earlier than previous executions of the module, the older log files will be ignored and not ingested into Wazuh.
+
+On the other hand, the ``CloudWatch Logs`` module can process logs older than the first one processed. To do that it is only necessary to specify an older ``only_logs_after`` value, and the module will process all logs between the value set for ``only_logs_after`` and the first log executed without generating duplicate alerts.
+
+
+Reparse
+~~~~~~~
+
+.. note::
+  Option not available for CloudWatch Logs.
+
+.. warning::
+  Using the ``reparse`` option will fetch and process every log from the beginning date until the present. It will most likely generate duplicate alerts.
```

Comment:
Add a link to the reparse reference

Reply:
There is no reparse reference and I think that we shouldn't include it since that parameter can't -and shouldn't- be used within the `ossec.conf` file configuration.